src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -455,11 +440,7 @@ private void init(long recordOffset) {
       // recover will only check pipe status and create pipe if it does not exist.
       recover(pipeCreation);
 
-      // when exactly_once is enabled,fetch clientSequencer and offsetPersistedInSnowflake
-      if (ingestionDeliveryGuarantee
-          == SnowflakeSinkConnectorConfig.IngestionDeliveryGuarantee.EXACTLY_ONCE) {
-        initClientInfoForExactlyOnceDelivery();
-      }
+      initClientInfo();
```

Comment:
the whole idea behind snowpipe's exactly once api was to use streaming channel like persistence layer behind a snowpipe. 
TLDR: Store offset tokens in pipe's metadata using invisible channel object on server side. (Not putting a lot of information here since this is public repo)

Lets remove all occurrences of clientInfo and calling clientConfigure API. We will not need it.

Reply:
Lets discuss this during our 1:1 tomorrow, I dont have enough context to make these changes currently